.github/workflows/calibration_test.yml

```diff
@@ -0,0 +1,22 @@
+name: Calibration
+on:
+  push:
+    tags: '*'
+  pull_request:
+  merge_group:
+
+jobs:
+  test:
+    name: ClimaCalibrate E2E Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: julia-actions/cache@v2
```

Comment:
If you add `julia-actions/cache`, you need to add the permission to read/write the cache. 

Also, it's a good idea to add a concurrency group so that new runs kill previous ones.

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

# Needed to allow julia-actions/cache to delete old caches that it has created
permissions:
  actions: write
  contents: read
```